core/src/main/java/com/radixdlt/consensus/bft/VertexStoreJavaImpl.java

```diff
@@ -250,7 +234,31 @@ public boolean insertTimeoutCertificate(TimeoutCertificate timeoutCertificate) {
   // TODO: reimplement in async way
   @Override
```

Comment:
Maybe it's worth a comment here (and/or in start-up) explaining why we execute lazily? 👍

Reply:
:+1: added a comment

Reply:
Looks good! But can we make the comment an actual Java Doc, so it appears in peoples' IDEs if they hover over the method?